agimus_controller_ros/launch/agimus_controller.launch.py

```diff
@@ -0,0 +1,27 @@
+from launch import LaunchDescription
+from launch.substitutions import PathJoinSubstitution
+
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+
+def generate_launch_description():
+    mpc_params = PathJoinSubstitution(
+        [
+            FindPackageShare("agimus_demo_01_lfc_alone"),
+            "config",
+            "linear_feedback_controller.yaml",
+        ]
+    )
```

Comment:
@MaximilienNaveau should agimus controller be aware or LFC params? I think if those two should share parameters then this should be handled with one file that is shared between them with a minimal subset of parameters.